UITests/run.py

```diff
@@ -61,8 +85,37 @@
 destination_arg = '-destination'
 
 for device in devices:
+	# get the simulator's uuid
+	list = subprocess.Popen('xcrun simctl list devices 13.5'.split(), stdout=subprocess.PIPE)
+	device_grep = subprocess.Popen(['grep', '{} ('.format(device)], stdin=list.stdout, stdout=subprocess.PIPE)
+	list.stdout.close()
+	uuid_grep = subprocess.Popen('grep -E -o -i ([0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12})'.split(), stdin=device_grep.stdout, stdout=subprocess.PIPE)
+	device_grep.stdout.close()
+	simulator_uuid = uuid_grep.communicate()[0].decode().strip()
+
+	simulator_fontSize_preferences_file = '{}/Library/Developer/CoreSimulator/Devices/{}/data/Library/Preferences/com.apple.UIKit.plist'.format(os.path.expanduser("~"), simulator_uuid)
+	if os.path.exists(simulator_fontSize_preferences_file):
+		# shutdown the simulator in case it is running
+		print("Shutdown the {} simulator in case it is running".format(device))
+		subprocess.call(["xcrun", "simctl", "shutdown", device], stderr=open(os.devnull))
+
+		# set the simulator's font size
+		print("Set the {} simulator\'s font size to {}".format(device, font_size))
+		simulator_largerText_preferences_file = '{}/Library/Developer/CoreSimulator/Devices/{}/data/Library/Preferences/com.apple.preferences-framework.plist'.format(os.path.expanduser("~"), simulator_uuid)
+		plistlib.writePlist({'largeTextUsesExtendedRange': True}, simulator_largerText_preferences_file) # enable larger text
+
+		subprocess.call(['plutil', '-replace', 'UIPreferredContentSizeCategoryName',
+		'-string', font_size, simulator_fontSize_preferences_file]) # set the size
+
+		# boot the simulator
+		print("Boot the {} simulator".format(device))
+		subprocess.call(["xcrun", "simctl", "boot", device])
+	else:
+		print("Unable to set font size on the {} simulator. This happen if it's the first time you boot it, try it again.".format(device))
```

Comment:
Unfortunately this could happen because the file needed to change the simulator's font size: ```~/Library/Developer/CoreSimulator/Devices/<device uuid>/data/Library/Preferences/com.apple.UIKit.plist```
is generated the first time the simulator is booted.